vervet-underground/server.go

```diff
@@ -22,8 +22,10 @@ import (
 
 func main() {
 	var wait time.Duration
+	var interval time.Duration
```

Comment:
Might be nice to have more consistency between the var and flag names. 

```suggestion
	var scrapeInterval time.Duration
```

and likewise for the other options.

Reply:
Another thought: single-source these options and 12-factor app it in the process with https://github.com/spf13/viper?

Reply:
@cmars I think making the options nicer would probably go through cobra instead to make them flag OR config. That being said, I'd like to avoid mixing that in here.